lib/index.js

```diff
@@ -59,9 +59,13 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         // since anyone can request a nonce,
         // this is permitted without authentication
         const {actor = null} = (req.user || {});
+        let typeOptions;
+        if(authenticationMethod === 'token-client-registration') {
+          typeOptions = {entryStyle: 'machine'};
+        }
```

Comment:
We should just be able to specify `typeOptions` via the client calling the API, right? Not hard code this here for a particular `authenticationMethod`?

Reply:
Updated to get typeOptions from client.